0011-reporting.md

```diff
@@ -0,0 +1,195 @@
+# Introduce reporting functionality
+
+* Status: proposed
+* Deciders: Jesús García Crespo, Sarah Romkey, Ross Spencer, Joel Simpson, Peter Van Garderen
+* Date: 2020-05-19
+
+## Context and problem statement
+Archivematica users want statistics about what their Archivematica deployments are doing and when, as well as detailed breakdowns of the content in their Archivematica systems. While the existing Archival Storage search and hit display does provide some useful output, it does not aggregate this information or present it in management-style reports. Nor does it provide this information in a machine readable format that can later be consumed by other tools or Archivematica itself.
+
+Development on a comprehensive reporting feature has been blocked in the past because it hasn't been clear where the canonical source of Archivematica statistical and content information is stored or which of these sources is the most persistent, convenient, durable, comprehensive, or performant input for building reports. This raises the question whether reporting should be a functionality supported natively by the Archivematica application or externally by a third-party tool, be that bespoke and developed by Artefactual, or off-the-shelf.
+
+There is also some overlap of requirements between logging functionality, reporting features, and the outputs provided by Archivematica's Format Policy Registry (FPR) which makes the final vision for an Archivematica reporting solution less clear.
+
+The functional output of reporting should be to support preservation planning, system implementation, and repository maintenance. We recognize that there are gaps in the data aggregated by Archivematica, e.g. technical properties across the widest range of file-formats; or the data available to systems like Archivematica, e.g. community driven statistics describing file-format obsolescence.
```

Comment:
```
The functional output of reporting should be to support preservation planning, 
system implementation, and repository maintenance.
```
I think I wrote this confusingly the first time around, but rather than system implementation, I think `implementation of preservation plans` would more accurate here.

Reply:
I pushed this revision to the branch.